packages/teleport/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@gravitational/design": "1.0.0",
     "@gravitational/shared": "1.0.0",
+    "yaml": "^2.2.0",
```

Comment:
I'm on the fence about including this, just because it's probably a heavy dependency and the backend has to verify the YAML anyway.

Reply:
@zmb3 Does it mean you'd rather do that on the backend?

Reply:
We could include the resource name as parameter in the API endpoint so that the validation can be done on the server.
Something like this:
- client: https://github.com/gravitational/webapps/commit/6adee1f637ab62fac8ede1578421c12e84b9d565
- server: https://github.com/gravitational/teleport/commit/f85288ad24572198f1f8fd18fb11ae0bf46ee251

WDYT?

Reply:
That looks better to me, what do you think @zmb3?

Reply:
I would go checking this server-side so that all validation is done on the server (and no new dependency).

@ravicious @zmb3 If you're ok with this, I'll wait for the webapps repo merge so that we don't have to change 4 repos.